dcs-dtc/UI/Aircrafts/AH64/UploadToJetPage.cs

```diff
@@ -0,0 +1,97 @@
+﻿using DTC.Models;
+using DTC.Models.Base;
+using DTC.Models.AH64;
+using DTC.UI.Base.GlobalHotKey;
+using DTC.UI.CommonPages;
+using System;
+
+namespace DTC.UI.Aircrafts.AH64
+{
+    public partial class UploadToJetPage : AircraftSettingPage
+    {
+        private AH64Upload _jetInterface;
+        private readonly AH64Configuration _cfg;
+
+        private KeyboardHookManager _keyboardHookManager;
+        public UploadToJetPage(AircraftPage parent, AH64Configuration cfg) : base(parent)
+        {
+            InitializeComponent();
+            _jetInterface = new AH64Upload(cfg);
+
+            txtWaypointStart.LostFocus += TxtWaypointStart_LostFocus;
+            txtWaypointEnd.LostFocus += TxtWaypointEnd_LostFocus;
+            txtWaypointStart.Text = cfg.Waypoints.SteerpointStart.ToString();
+            txtWaypointEnd.Text = cfg.Waypoints.SteerpointEnd.ToString();
+            _cfg = cfg;
+
+            chkWaypoints.Checked = _cfg.Waypoints.EnableUpload;
+            chkRadios.Checked = _cfg.Radios.EnableUpload;
+
+            CheckUploadButtonEnabled();
+
+            _keyboardHookManager = new KeyboardHookManager();
+            _keyboardHookManager.Start();
+
+            HotKey hotkey;
+            if (ParseHotKey.TryParse(Settings.UploadHotKey, out hotkey))
+            {
+                _keyboardHookManager.RegisterHotkey(hotkey.Modifiers, (int)hotkey.Key, () =>
+                {
+                    _jetInterface.Load();
+                });
+            }
+        }
+        private void CheckUploadButtonEnabled()
+        {
+            btnUpload.Enabled = (_cfg.Waypoints.EnableUpload || _cfg.Radios.EnableUpload);
+        }
+
+        public override string GetPageTitle()
+        {
+            return "Upload to Heli";

```

Comment:
Maybe the filename should also be changed to UploadToHeliPage or so

Reply:
Changed to UploadToHeliPage